app/main/lib/shared_models/video_model.py

```diff
@@ -155,21 +160,22 @@ def search(self, task):
                             "score": score,
                             "model": "video"
                         })
-                if temporary:
-                    self.delete(task)
                 limit = task.get("limit")
                 if limit:
                     return {"result": results[:limit]}
                 else:
                     return {"result": results}
             else:
-                ErrorLog.notify(Exception("Failed to locate needle for a video!"), {"video_folder": video.folder, "video_filepath": video.filepath, "video_id": video.id, "task": task})
                 return {"error": "Video not found for provided task", "task": task}
-        else:
-            return {"error": "Video not found for provided task", "task": task}
+        except Exception as err:
+            raise err
```

Comment:
Can we (do we need to) send to Sentry explicitly here? I think we should log in sentry if we're not certain that happens already

Reply:
Lets just add it for now - easy to see if we're double-logging and revert if necessary

Reply:
Tweaked this commit so that we're not passing `video` attributes along since `video` wouldn't be guaranteed at this point of execution